templates/boltzmann/core/entrypoint.ts

```diff
@@ -21,6 +28,11 @@ if (require.main === module && !process.env.TAP) {
     return (next: Handler) => (context: Context) => next(context)
   }
 
+  // {% if honeycomb %}
+  initOtelSDK().then(run)
+  function run() {
```

Comment:
Getting a little tricky here. If the honeycomb feature is on, this runserver call is wrapped in a function; if it's off, the call happens directly. It *might* be more confusing than it's worth, but I'm tickled by it and think it's consistent with our use of templating in boltzmann, but definitely keep me honest!